src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -96,11 +103,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     }
 
     UUID conversationId = conversation.getId();
+    List<UUID> members = conversation.getMembers();
+    List<User> userList = new ArrayList<User>();
+    for(UUID member : members){
+    	userList.add(conversation.getUserName(member));
```

Comment:
I think since this class already has a userStore variable, this line could just call userStore.getUser(member) instead of passing it to Conversation to do the same thing. Then we don't need that new method inside Conversation.